node/src/components/linear_chain.rs

```diff
@@ -110,7 +110,7 @@ struct SignatureCache {
 impl SignatureCache {
     fn new() -> Self {
         SignatureCache {
-            curr_era: EraId(0),
+            curr_era: EraId::from(0),
```

Comment:
Maybe we should even add an `EraId::genesis()` or `EraId::zero()` constructor or constant.